public/scripts/keyGen.js

```diff
@@ -0,0 +1,82 @@
+const ALPHANUMERIC_CAPITALS = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+let keysBox = document.querySelector('#keys_box');
+let formBox = document.querySelector('#form_box');
+let code_input = document.querySelector('#code_input');
+let code_button = document.querySelector('#code_button');
+
+if (keysBox.childElementCount > 0) {
+    formBox.children[0].children[1].removeChild(code_input);
+    formBox.children[0].children[1].removeChild(code_button);
+}
+
+function generateKey(id) {
+    let input = parseInt(document.querySelector('#participants_number').value);
+    let errorMsg = document.querySelector('#validation_msg');
+    if (errorMsg !== null) {
+        formBox.removeChild(errorMsg)
+    }
+
+    if (input === undefined || input < 1 || input > 50 || !Number.isInteger(input)) {
```

Comment:
```suggestion
    if (!input  || input < 1 || input > 50) {
```
Volgensmij doet dit hetzelfde

Reply:
De laatste controleert specifiek dat je geen string value oid injecteert
Dus ze doen wel zeker iets anders

Reply:
Als er een waarde uit een htmlElement wordt gehaald is het standaard een `string`, daarom wordt er vervolgens `parseInt` gedaan. De functie `parseInt` verandert een string naar een getal of `Nan`. Aangezien `Nan` ook wordt gezien als `false`, kan je dus controleren of de waarde van de input gelijk is aan false, door dit te doen: `!input`

Reply:
Hier kan je het eens uitproberen: https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_parseint